cms/static/js/factories/program_editor.js

```diff
@@ -0,0 +1,9 @@
+define([
+    'js/programs/program_admin_app'
+],
+function(program_admin_app) {
+  'use strict';
+  return function(){
```

Comment:
Nits: this should use four spaces for indentation, and there should be a space before the bracket:

    return function() {